src/main/java/by/custom/utilcalculator/service/CalculatorPassenger.java

```diff
@@ -26,10 +28,11 @@ public String calculate(CountryOrigin countryOrigin, OwnersType ownersType, Type
             case null -> {
                 return "calculator: countryOrigin is null";
             }
-            case default -> {
+            default -> {
                 return "calculator: calculate() unknown error";
             }
         }
+//        return "calculator: calculate() unknown error";
```

Comment:
why not remove it?

Reply:
I love this messages a lot. They really helped me when I changed Java-version, and it was hard for me to remove it. Anyway, new commit doesn't contain this comments. Sorry for this mess. I will try to put my code in better order.